cluster-scope/overlays/prod/moc/smaug/kustomization.yaml

```diff
@@ -27,7 +33,6 @@ resources:
 - ../../../../base/config.openshift.io/oauths/cluster
 - ../../../../base/core/configmaps/cluster-monitoring-config
 - ../../../../base/core/configmaps/user-workload-monitoring-config
-- ../../../../base/core/namespaces/acme-operator
```

Comment:
Have you also checked the other kustomizations? `smaug` is not the only cluster that uses these manifests from the `base` directory. 

`acme-operator` for instance is also used by the osc cluster, so it's included in the osc overlay [here](https://github.com/operate-first/apps/blob/master/cluster-scope/overlays/prod/osc/osc-cl1/kustomization.yaml).

Reply:
You should cross check `diffs` from  buildling all the kustomizations that occur in `apps/cluster-scope/overlays/prod/$ENV/$CLUSTER/`

Reply:
Thanks for taking a look! In this PR, I'm only doing bundling for smaug cluster. I will be creating more PRs to send in the changes for other clusters.

Reply:
kk sounds good!